source/agora/consensus/EnrollmentManager.d

```diff
@@ -242,14 +241,16 @@ public class EnrollmentManager
         Params:
             frozen_utxo_hash = the hash of a frozen UTXO used to identify a validator
                         and to generate a siging key
+            current_height = the starting index for generating pre-images
```

Comment:
But this is not the starting block. It is the height of the current ledger. If the enrollment is ever added to the blockchain, it will be at a future unspecified block height.

Reply:
> But this is not the starting block. It is the height of the current ledger. If the enrollment is ever added to the blockchain, it will be at a future unspecified block height.

It's a starting height for generating pre-images, not a starting height as validator.